pkg/cmd/scan.go

```diff
@@ -231,9 +234,12 @@ func scanRun(opts *pkg.ScanOptions) error {
 		return err
 	}
 
-	err = selectedOutput.Write(analysis)
-	if err != nil {
-		return err
+	for _, o := range opts.Output {
+		selectedOutput := output.GetOutput(o, opts.Quiet)
+		err = selectedOutput.Write(analysis)
+		if err != nil {
+			return err
```

Comment:
I am wondering if we should break on the first error or if we should continue and log an error message. WDYT @wbeuil @moadibfr ? Compared on how we'll handle `--from` flags after @moadibfr's work, that may be more consistent not to break on first error.

Reply:
I would recommend your last remark which is to not break on first error. We could display an alert.

Reply:
Sending an alert there is impossible so I added an error log. Also added a fallback to console output when all outputs failed to compute.